lib/ta_profile_page.dart

```diff
@@ -28,6 +29,14 @@ class _TaProfilePageState extends State<TaProfilePage> {
     setState(() {});
   }
 
+  selectedCourse(Course course) async {
```

Comment:
`selectCourse` because it's kinda action, isn't it?

Reply:
True. Fixed.